bundles/org.smarthomej.binding.notificationsforfiretv/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="notificationsforfiretv"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="notification">
+		<label>Notifications for Fire TV</label>
+		<description>Notifications for Fire TV Binding</description>
+
+		<config-description>
+			<parameter name="ip" type="text" required="true">
+				<context>network-address</context>
+				<label>IP Address</label>
+				<description>Device ip address.</description>
+			</parameter>
+			<parameter name="port" type="integer" required="true" min="0" max="65535">
+				<label>Port</label>
+				<description>Device port listening to connections.</description>
+				<default>7676</default>
+			</parameter>
+			<!-- NO EFFECT parameter name="type" type="integer" required="false" min="0" max="65535">
```

Comment:
If it can't be used, it ashpuld probably be removed,

Reply:
done, removed it